### PR TITLE
[2.5pt] - Generate Reservation Model & Controller#6

### PR DESCRIPTION
**Pull request by Dachrono**
# 🎉 🎈 In this *Milestone* 🙋‍♂️ 🚩 ‼ :

**Hi there** 👋

- Follow the best GitHub flow. ✅
- Follow our list of best practices for Ruby on Rails. ✅
- Document in a professional way. ✅
- No linters errors. ✅
- Create a reservation controller. ✅
- Create a reservation model. ✅

# To the code reviewer 👨‍💻
I really thank 🙇‍♂️ you, dear code reviewer 👨‍🎨 for dedicating your time ⌚ to check the PR 🧾 .

I kindly ask code reviewer 🤵 that if there is any issue in this project, please do list 📃 them in a descriptive 💡 manner and give your best suggestions if needed.

If you think some significant issues need to be changed ♻️ please kindly contact 📞 me through Zoom or Slack, I am available on Microverse UTC-6 time zone.

### CODE REVIEW

File: app/controllers/api/v1/reservations_controller.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class ReservationsController < ApplicationController
+  def index
```
Comment (HFG43) [inline app/controllers/api/v1/reservations_controller.rb:2]: Great Job @Dachrono including the controller in Version 1 of the API. In order of the controller, to have the correct access to the Application Controller, it should be called using the corresponding folder path. In this case, should follow the following naming: `class Api::V1::ReservationsController < ApplicationController`

File: app/controllers/api/v1/reservations_controller.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    reservations = Reservation.all
+    render json: reservations
+  end
```
Comment (HFG43) [inline app/controllers/api/v1/reservations_controller.rb:5]: Hi @Dachrono Great job implementing the index and delete actions for reservations. Please consider, that for this part of the app, we should present the user reservations list, so you will need the user as part of the index

**Review by HFG43 — CHANGES_REQUESTED** [review]
Hi @Dachrono !! Great job so far! Just some adjustments are needed in the `reservations` controller in order to make sure to access and display the corresponding user reservations.

With these changes, your Milestone will be ready to merge!

File: app/controllers/api/v1/reservations_controller.rb (hunk 1 above)
Comment (Dachrono) [reply app/controllers/api/v1/reservations_controller.rb:2]: Tky for the comments, Solved

**Review by HFG43 — APPROVED** [review]
Great job @Dachrono implementing the changes!
Excellent work on this Milestone!!

There is nothing else to say than this branch is ready to merge!!

**Pull request merged**